tests/integration_test_net.go

```diff
@@ -70,6 +71,9 @@ type IntegrationTestNetOptions struct {
 	NumNodes int
 	// ClientExtraArguments specifies additional arguments to be passed to the client.
 	ClientExtraArguments []string
+	// GenesisCommittee specifies the genesis committee to be used for the integration
+	// test network. If not set, a default committee is used.
+	GenesisCommittee scc.Committee
```

Comment:
This can not be an option of the integration test network. The problem is that it would not be possible for the network nodes in the test network to figure out the private keys to act as the members of this committee.

Instead, the fake genesis is supposed to define a committee, start up the network using this committee, and the integration tests should retrieve the chosen genesis committee from the integration test network.

Reply:
ok, I will drop the changes for the integration test net, and focus on the light client only for this PR

Reply:
https://github.com/0xsoniclabs/sonic/pull/120/commits/a6b4c6fec6b9b98d8f7358cecbed3f890e070ac8